config/rollup.browser.config.js

```diff
@@ -0,0 +1,18 @@
+import fs from "fs";
+import config from './rollup.iife.config';
+import { builtinModules as builtin } from "module";
+
+const pkg = JSON.parse(fs.readFileSync("./package.json"));
+
+export default {
+    ...config,
+    output: {
+        ...config.output,
+        file: "build/snarkjs.js",
```

Comment:
```suggestion
        file: "build/snarkjs.esm.js",
```
We can make this `.esm.js` so the iife path doesn't change.